bin/sync_vagrant_builds.rb

```diff
@@ -0,0 +1,102 @@
+#!/bin/env ruby
```

Comment:
This doesn't work on a Mac...can we make it 
```suggestion
#!/usr/bin/env ruby
```